### ISSUE TITLE
how to install this module

### ISSUE THREAD

**Issue by mominurr** [pcorpet/airtable.py#63]
I am cloning this module but how to import this module? I am not finding any airtable.py file .

**Comment by pcorpet** [comment]
You're right, the `airtable` library is structured as a *package*, not a single module.  That's why there's an `airtable/__init__.py` file. This file (even if empty) signals to Python that the `airtable` directory should be treated as a package, allowing for more organized code.

Here's how to import it:

```python
import airtable
```

This imports the entire `airtable` package.  You can then access its contents (classes, functions, etc.) using dot notation, e.g., `airtable.Airtable(...)` (assuming `Airtable` is a class defined within the package).

**Key Difference: Module vs. Package**

*   A **module** is a single Python file (e.g., `my_module.py`). You import it directly: `import my_module`.
*   A **package** is a collection of modules (and potentially sub-packages) within a directory.  The directory *must* contain an `__init__.py` file (even if it's empty) to be recognized as a package. You import the package name: `import my_package`.

Packages help organize larger projects into logical units.  See the official Python documentation for more details: [[https://docs.python.org/3/tutorial/modules.html#packages](https://www.google.com/search?q=https://docs.python.org/3/tutorial/modules.html%23packages)](https://www.google.com/search?q=https://docs.python.org/3/tutorial/modules.html%23packages)
